YaLibs/YaToolsIDALib/Repository.cpp

```diff
@@ -765,23 +778,13 @@ bool Repository::rebase(const std::string& upstream, const std::string& destinat
 
 bool Repository::add_file_to_index(const std::string& path)
 {
-    fs::path p = path;
-    if (!include_idb_ &&
-        (path == get_current_idb_name() ||
-        path == get_original_idb_name()))
-    {
-        p = ".gitignore";
-        std::fstream f;
-        f.open(p, std::fstream::out);
-        f << get_current_idb_path() << std::endl;
-    }
     try
     {
-        repo_.add_file(p.generic_string());
+        repo_.add_file(path);
```

Comment:
Wasn't this a problem on Windows before?

Reply:
fs::path p was removed, because we don't need to manipulate it
std::string path is used instead like before

Reply:
Didn't notice that  change.  Everything lgtm.